sorc/ncep_post.fd/ALLOCATE_ALL.f

```diff
@@ -79,7 +79,7 @@ SUBROUTINE ALLOCATE_ALL()
 !$omp parallel do private(i,j,l)
       do l=1,lm
         do j=jsta_2l,jend_2u
-          do i=1,im
+          do i=1,lm
```

Comment:
@EdwardColon-NOAA May I ask the reason of  changing im into lm?

Reply:
@EdwardColon-NOAA    I have the same questions for reason of this changing im to lm. Ed you might have mentioned and explained this issue before in our internal meeting. I forgot it.

Reply:
The i loop min.max indices were mistakenly altered when updating the code. My apologies. This problem has been corrected.